components/courses/app/models/courses/test_task.rb

```diff
@@ -1,7 +1,7 @@
 module Courses
   class TestTask < ApplicationRecord
     self.table_name = 'courses_test_tasks'
-    belongs_to :student, optional: true
+    belongs_to :student
```

Comment:
How about a situation when a mentor creates a new interview slot, which doesn't belong to a student yet.